dependencies.gradle

```diff
@@ -2,7 +2,7 @@
  * Common place to define all the version dependencies
  */
 ext {
-    revConductor = '+'
+    revConductor = '3.13.8'
```

Comment:
We need to now pin the version to before 3.14. Conductor was recently updated to use Java 17. We're not using it directly but because it checked the dependency as incompatible version, it fails the build.